src/debugger.ts

```diff
@@ -114,6 +123,10 @@ export namespace Debugger {
      * The sources widget.
      */
     readonly sources: Sources;
+
+    readonly layoutSplit: SplitPanel;
```

Comment:
I would suggest calling this `readonly body` instead of `readonly layoutSplit` since it's more semantically clear.

Reply:
Also, does it need to be public, or event an attribute to the class?